docs/_articles/en/functions/overview.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Overview
+lang: en
+---
+## Salesforce Functions (Beta)
+Use the Salesforce Functions extensions in VSCode to build event-driven, elastically scalable apps and experiences.
+
+Use the Salesforce features for VS Code to create and test Salesforce Functions locally.
+
+### Create Function
+Run `SFDX: Create Function` to create a Salesforce Function in Javascript or Java in a Salesforce DX project.
+
+A new directory named after your Function name and all the supporting files are created for you.
+
+* VS Code creates the basic scaffolding that contains files with some rudimentary, boilerplate code and supporting metadata.
+* The `package.json` file contains information about dependencies.
+
+![Create Function](../../../images/vs_code_create_func.gif)
+### Start Function
+Run `SFDX: Start Function` from the command palette to run the function locally.
```

Comment:
You don't say "from the command palette" in Create Function. Do we think that's implied at this point? That all the VS code commands are "from" the palette.